index.html

```diff
@@ -9,29 +9,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 
-
+    <link rel="stylesheet" href="./style.css">
 </head>
 
 <body>
-    <ul>
-
-        <li><a href="./components/avatar/avatar.html">Avatar</a></li>
-        <li> <a href="./components/alert/alert.html">Alert </a></li>
-        <li> <a href="./components/badge/badge.html">Badge </a></li>
-        <li> <a href="./components/button/button.html">Buttons </a></li>
-        <li> <a href="./components/cards/cards.html">Cards</a></li>
-        <li> <a href="./components/image/image.html">Image</a></li>
-        <li> <a href="./components/input/input.html">Input</a></li>
-        <li> <a href="./components/text-utilities/text.html">Text utilities</a></li>
-        <li> <a href="./components/navigation/navigation.html">Navigation</a></li>
-        <li> <a href="./components/list/list.html">List</a></li>
-    </ul>
+    <nav class="header-nav">
+        <div class="brand">
+
+            <a href="#">
+                <h1 class="h-2">Nectar UI</h1>
+            </a>
+        </div>
+
+        <div class="actions">
+            <a href="./components/alert/alert.html"><button class="btn-cta">Docs</button></a>
+            <a href="https://github.com/suryxks/Nectar-ui" target="_blank"><img src="/assets/github.svg"></a>
+        </div>
+    </nav>
+    <section class="hero">
+        <div>
+
+            <h1 class="h-1">Build Beautiful and Performant websites using <span class="brand">Nectar UI</span></h1>
+            <p class="text-md">Nectar is a CSS library with pre-defined styled classes and Utilities to get you quickly
+                started with your project</p>
+            <div class="actions-hero">
+                <a href="./components/alert/alert.html"><button class="btn-cta">Docs</button></a>
+                <a href="https://github.com/suryxks/Nectar-ui" target="_blank"><button
+                        class="btn-cta-outlined">Github</button></a>
+            </div>
+        </div>
+        <img src="./assets/hero.svg" class="hero-img">
+
+
+    </section>
+    <div class="installation display">
+        <div>
+            <h1 class="h-1">Installation</h1>
+            <p class="text-md">Copy and paste the code in the head tag of your html document. So that you can style your
```

Comment:
Can we create more descriptive classes instead of "h-1" say "large heading"

Reply:
Thank you for the feedback poornima ,i have made the changes and will merge the PR